packages/react-ui/src/widgets/CategoryWidgetUI/CategoryWidgetUI.js

```diff
@@ -432,22 +432,19 @@ function CategoryWidgetUI(props) {
     );
   };
 
-  const CategoryRow = useCallback(
```

Comment:
Couldn't removing `useCallback` cause unnecessary re-renders in the virtualized list?

Reply:
You're just rendering up to 9 elements, so it's not going to cause huge performance issues 👍

Reply:
ok, thanks! please consider adding someone from Builder to the review, as they probably have more context about this issue.

Reply:
Sure! 👍